api/src/pcapi/core/offerers/api.py

```diff
@@ -1965,8 +1965,18 @@ def accept_offerer_invitation_if_exists(user: users_models.User) -> None:
         user_offerer = models.UserOfferer(
             offerer=offerer_invitation.offerer, user=user, validationStatus=ValidationStatus.NEW
         )
+        extra_data = {"inviter_user_id": inviter_user.id, "offerer_invitation_id": offerer_invitation.id}
+        log_action = history_api.log_action(
+            history_models.ActionType.USER_OFFERER_NEW,
+            user,
+            user=user,
+            offerer=offerer_invitation.offerer,
+            save=False,
+            comment="Invitation de rattachement acceptée",
```

Comment:
```suggestion
            comment="Rattachement créé par invitation",
```
Mais c'est une suggestion seulement